src/components/team/account-ownership-transfer.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react'
+import toast from 'react-hot-toast'
+import {AUTH_DOMAIN, getAuthorizationHeader} from '../../utils/auth'
+import axios from 'axios'
+
+const sendOwnershipTransferInvite = async (
+  inviteeEmail: string,
+  accountId: number | undefined,
+) => {
+  const account_id = accountId || ''
+
+  return axios.post(
+    `${AUTH_DOMAIN}/api/v1/account_ownership_transfer_invitations`,
+    {account_id, invitee_email: inviteeEmail},
+    {
+      headers: {...getAuthorizationHeader()},
+    },
+  )
+}
+
+const AccountOwnershipTransfer = ({accountId}: {accountId: number}) => {
+  const [loading, setLoading] = React.useState<boolean>(false)
+  const [inviteeEmail, setInviteeEmail] = React.useState<string>('')
+
+  return (
+    <>
+      <h2 className="font-semibold text-xl mt-16">
+        Account Ownership Transfer
+      </h2>
+      <p>
+        You are current the owner of this egghead team account. You can transfer
+        that ownership to another egghead user. Submit their egghead user email
+        address with this form and we will email them an invite. Once they
+        accept the invite, ownership of this team account will be tranferred
+        from you to them.
+      </p>
+      <div className="flex flex-col space-y-2 mt-6">
+        <input
+          className="bg-gray-50 dark:bg-gray-800 focus:outline-none focus:shadow-outline border border-gray-300 dark:border-gray-700 rounded-md py-2 px-4 block w-full sm:w-1/2 md:w-1/3 appearance-none leading-normal"
+          type="email"
+          value={inviteeEmail}
+          onChange={(e) => setInviteeEmail(e.target.value)}
+          placeholder="Invitee Email"
+        />
+        <div className="flex flex-row space-x-2">
+          <button
+            className={`text-white bg-green-600 border-0 py-2 px-4 focus:outline-none rounded-md
+                    ${
+                      loading || inviteeEmail === ''
+                        ? 'cursor-not-allowed opacity-50'
+                        : 'hover:bg-green-700'
+                    }`}
+            type="button"
+            disabled={loading || inviteeEmail === ''}
+            onClick={async () => {
+              try {
+                setLoading(true)
+
+                await sendOwnershipTransferInvite(inviteeEmail, accountId)
+
+                setInviteeEmail('')
+
+                toast.success(
+                  'Your account ownership transfer invitation has been sent.',
+                  {
+                    icon: '✅',
+                  },
+                )
+              } catch (e) {
+                toast.error(
```

Comment:
when we start digging into these kinds of errors, how should we capture them on our side?

A honeybadger project might help. We had bad luck with their autocapture, but manually it could provide insight into business processes gone wrong.

Reply:
I'm adding Ahoy tracking calls for both the successful and failed invite requests.

It would be nice to have a more dedicated error capture approach setup. I had assumed that Honeybadger was probably already being used throughout this codebase, but I did a search and see that it isn't.

We can start capturing specific things like this using the Honeybadger JS lib and see how that works for errors in these kinds of business processes.

I agree that we should use a separate Honeybadger project for this.